src/fides/api/ctl/database/seed.py

```diff
@@ -278,6 +296,46 @@ async def load_default_dsr_policies() -> None:
             except KeyOrNameAlreadyExists:
                 # This rule target already exists against the Policy
                 pass
+    else:
+        log.info(
+            f"Skipping {DEFAULT_ERASURE_POLICY_RULE} creation as it already exists in the database"
+        )
+
+
+async def load_default_dsr_policies() -> None:
```

Comment:
There is no `await` so this should be a `def` instead of `async def`. Or should it be using the `async_session`?

Reply:
not sure on the async session, it had never been using it before so i'd prefer not to change that in this PR.

i'll update to remove the `async`, i was also confused why it was `async` but just decided to leave it alone. just to be clear, i didn't actually change the function signature (just changed the signature's spot in the file so it comes up here in the diff), but i'll make this update (assuming it passes tests) because i agree with you that it shouldn't be `async`